src/en/send.md

```diff
@@ -97,58 +97,100 @@ You can leave out this argument if your service only has one reply-to email addr
 Sending files by email is not enabled by default. To turn on this feature, [contact us](https://notification.canada.ca/contact).
 :::
 
-To send a file by email, you'll need to add a placeholder to the template then upload a file. The placeholder will contain a secure link to download the file. 
+### File types and size requirements
+You can upload PDF, CSV, .jpeg, .png, .odt, .txt, .rtf,  Microsoft Excel and Microsoft Word Document files. Your file must be smaller than 10MB.
+
+[Contact us](https://notification.canada.ca/contact) if you need to send other file types.
+
+
+### Sending methods
+You can send files in two ways on Notify:
+
+1. directly attached to emails
+1. through a unique link to a web interface. Links are unique and unguessable.
```

Comment:
Consider removing "Links are unique and unguessable"

Reply:
2. through a private link to a web interface.

Reply:
@dorianjp You don't need to number items from lists in Markdown, that's a useful feature.

You can do
```
1. One
1. Two
1. Three
```

and it will be rendered as

1. One
1. Two
1. Three